src/modules/logger/logged_topics.cpp

```diff
@@ -467,7 +467,7 @@ void LoggedTopics::add_mission_topic(const char *name, uint16_t interval_ms)
 bool LoggedTopics::add_topic(const orb_metadata *topic, uint16_t interval_ms, uint8_t instance, bool optional)
 {
 	if (_subscriptions.count >= MAX_TOPICS_NUM) {
-		PX4_WARN("Too many subscriptions, failed to add: %s %" PRIu8, topic->o_name, instance);
+		PX4_DEBUG("Too many subscriptions, failed to add: %s %" PRIu8, topic->o_name, instance);
```

Comment:
We should probably fix the root cause here instead. I'll take a look.

Reply:
We should, yes. But I don't know it...

Reply:
Any idea what to do @bkueng?

Reply:
You can either increase `MAX_TOPICS_NUM` which requires more stack, and/or remove some of the topics.

Reply:
Or ignore the warning? :D

Reply:
That's hardly a good solution. It means it will silently not log certain things.

Reply:
Ok, let me say this again: all I care about is not to see a warning, it's embarrassing if doing the default SITL startup we get spammed with warnings that aren't relevant.

Reply:
I'll trim down multi-ekf across all mags by default for now, at least until it factors in the configured priority.

Reply:
Ping.

Reply:
https://github.com/PX4/PX4-Autopilot/pull/21614

Reply:
Thanks, let me test this and rebase.